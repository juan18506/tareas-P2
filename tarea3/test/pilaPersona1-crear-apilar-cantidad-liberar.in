crearPila

imprimirCantidadPila

crearPersona 1 1 Persona1

apilar

liberarPersona

crearPersona 2 2 persona2

apilar

liberarPersona

imprimirCantidadPila

liberarPila

Fin