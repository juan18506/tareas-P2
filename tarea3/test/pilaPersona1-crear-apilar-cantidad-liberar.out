1>2>La cantidad de personas en la pila es: 0.
3>4>5>6>7>8>9>La cantidad de personas en la pila es: 2.
10>11>Fin.
